infra/metadata.yaml

```diff
@@ -41,125 +41,126 @@ spec:
+        description: Neos Tutorial URL
+      - name: server_service_name
+        description: Name of the Cloud Run service, hosting the server API
+      - name: usage
+        description: Next steps for usage
   requirements:
     roles:
-    - level: Project
-      roles:
-      - roles/cloudbuild.builds.editor
-      - roles/cloudsql.admin
-      - roles/compute.admin
-      - roles/compute.networkAdmin
-      - roles/firebase.managementServiceAgent
-      - roles/firebasehosting.admin
-      - roles/iam.serviceAccountAdmin
-      - roles/iam.serviceAccountUser
-      - roles/pubsub.editor
-      - roles/resourcemanager.projectIamAdmin
-      - roles/run.admin
-      - roles/secretmanager.admin
-      - roles/storage.admin
+      - level: Project
+        roles:
+          - roles/cloudbuild.builds.editor
+          - roles/cloudsql.admin
+          - roles/compute.admin
+          - roles/compute.networkAdmin
+          - roles/firebase.managementServiceAgent
+          - roles/firebasehosting.admin
+          - roles/iam.serviceAccountAdmin
+          - roles/iam.serviceAccountUser
+          - roles/pubsub.editor
+          - roles/resourcemanager.projectIamAdmin
+          - roles/run.admin
+          - roles/secretmanager.admin
+          - roles/storage.admin
     services:
-    - bigquery.googleapis.com
-    - bigquerymigration.googleapis.com
-    - bigquerystorage.googleapis.com
-    - cloudtrace.googleapis.com
-    - datastore.googleapis.com
-    - logging.googleapis.com
-    - monitoring.googleapis.com
-    - servicemanagement.googleapis.com
-    - serviceusage.googleapis.com
-    - sql-component.googleapis.com
-    - storage.googleapis.com
-    - storage-api.googleapis.com
-    - storage-component.googleapis.com
-    - artifactregistry.googleapis.com
-    - cloudbuild.googleapis.com
-    - cloudresourcemanager.googleapis.com
-    - compute.googleapis.com
-    - firebase.googleapis.com
-    - firebasehosting.googleapis.com
-    - iam.googleapis.com
-    - pubsub.googleapis.com
-    - run.googleapis.com
-    - secretmanager.googleapis.com
-    - sqladmin.googleapis.com
+      - bigquery.googleapis.com
+      - bigquerymigration.googleapis.com
+      - bigquerystorage.googleapis.com
+      - cloudtrace.googleapis.com
+      - datastore.googleapis.com
+      - logging.googleapis.com
+      - monitoring.googleapis.com
+      - servicemanagement.googleapis.com
+      - serviceusage.googleapis.com
+      - sql-component.googleapis.com
+      - storage.googleapis.com
+      - storage-api.googleapis.com
+      - storage-component.googleapis.com
+      - artifactregistry.googleapis.com
+      - cloudbuild.googleapis.com
+      - cloudresourcemanager.googleapis.com
+      - compute.googleapis.com
+      - firebase.googleapis.com
+      - firebasehosting.googleapis.com
+      - iam.googleapis.com
+      - pubsub.googleapis.com
+      - run.googleapis.com
+      - secretmanager.googleapis.com
+      - sqladmin.googleapis.com
```

Comment:
suggestion: Are these spacing changes something done by the CFT tooling or something else? If something else, reverting some of this may be worth considering to avoid file churn.

Reply:
this was from the `make` command